rpi-imager-haos.json

```
@@ -45,9 +45,25 @@
         "pi3-64bit"
       ]
     },
+    {
+      "name": "Home Assistant OS 14.0 (HA Yellow)",
```

I think I prefer just Yellow here, aligns better with the installer below (and technically that is the board name too) :sweat_smile: 

```suggestion
      "name": "Home Assistant OS 14.0 (Yellow)",
```